modules/docker/init.zsh

```diff
@@ -0,0 +1,35 @@
+#
+# Defines docker and boot2docker commands & aliases
+#
+# Authors:
+#   andrew williams (https://github.com/skarfacegc)
+#
+
+# Only setup the boot2docker commands and aliases if we're on OSX
+if [[ "$OSTYPE" == darwin* ]]; then
```

Comment:
Linux boxes dont since they can run docker natively  (don't need the virtualization layer that boot2docker provides).  Windows likely needs it, but I only have access to windows VMs so I can't test a vm within a vm.